templates/travis/.travis/script.sh.j2

```diff
@@ -13,6 +13,7 @@ export FUNC_TEST_SCRIPT=$TRAVIS_BUILD_DIR/.travis/func_test_script.sh
 # Gets set in .travis/settings.yml, but doesn't seem to inherited by
 # this script.
 export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
+export PULP_CONTENT_ORIGIN=http://$(hostname):24816
```

Comment:
This worked for me when I applied it to pulpcore, but I'm concerned about 2 things:

1) should this be guarded by the if == 'docs' below so it only applies to those builds which don't "inherit the settings"
2) should this be guarded by an if == 'pulpcore' to only set it on pulpcore?

Reply:
Yes to both.